test/mammoth.extensions.tests.js

```diff
@@ -50,8 +50,8 @@ test('formatted word tables are converted to html tables', function() {
     return mammoth.convertToHtml({path: docxPath}).then(function(result) {
         var expectedHtml = '<p>Above</p>' +
             '<table style="border-collapse: collapse;">' +
```

Comment:
Can you add individual test cases for our use case, i.e. 2 column table with non 50/50 distribution. And a 3 column table with non 50/50 distribution?

Reply:
Yup, I'll add that